pkgs/by-name/do/doomsday-engine/generic.nix

```diff
@@ -0,0 +1,49 @@
+{
+  gamekitPath,
+  ...
+}@drvArgs:
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+  cmake,
+  pkg-config,
+  python3,
+
+  SDL2,
+  SDL2_mixer,
+  the-foundation,
+  ncurses,
+}:
+stdenv.mkDerivation (drvArgs // {
+  nativeBuildInputs = [
+    cmake
```

Comment:
```suggestion
    cmake
    ninja
```
this could help us speed up builds a bit if it works

Reply:
I tried Ninja but it doesn't really seem to be faster? Lemme try running some benchmarks

Reply:
Ninja took 140 seconds and Make took 147 seconds with my experiments, which is not really a significant improvement IMO